examples/patterns/forms/forms.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Forms / Layout
+category: _patterns
+---
+
+<form class="p-form p-form--stacked u-float-left">
+  <div class="p-form__group">
+    <label for="full-name-stacked" class="p-form__label">Full name</label>
+    <div class="p-form__control">
+      <input type="text" id="full-name-stacked" required>
+    </div>
+  </div>
+  <div class="p-form__group p-form-validation is-error">
+    <label for="address-stacked" class="p-form__label">Address line 1</label>
+    <div class="p-form__control">
+      <input type="text" id="address-stacked" class="p-form-validation__input" required aria-invalid="true" aria-describedby="input-error-message-stacked">
+      <p class="p-form-validation__message" id="input-error-message-stacked" role="alert">
+        <strong>Error:</strong> Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+      </p>
+    </div>
+  </div>
+  <div class="p-form__group">
+    <label for="address-optional-stacked" class="p-form__label">Address line 2 (optional)</label>
+    <div class="p-form__control">
+      <input type="text" id="address-optional-stacked">
+    </div>
+  </div>
+  <button class="p-button--positive u-float-right">Add details</button>
+</form>
+<form class="p-form p-form--inline u-float-left">
```

Comment:
This should be in its own example and the docs should just have two examples with documentation.